test/test_models.py

```diff
@@ -157,23 +145,33 @@ def get_export_import_copy(m):
 
     sm = torch.jit.script(nn_module)
 
-    with torch.no_grad(), freeze_rng_state():
-        eager_out = nn_module(*args)
+    if eager_out is None:
+        with torch.no_grad(), freeze_rng_state():
+            if unwrapper:
```

Comment:
It's not mandatory to have it for most of the existing models but this is only due to implementation details. Since this is a general purpose tool for checking JIT-scriptability, I opted for consistently unwrapping the output in all place.

FYI the reason many models don't have to get unwrapped is due to idioms like this:
https://github.com/pytorch/vision/blob/4ae20e53a016b5d48ed8b4619499be3beb4b44f3/torchvision/models/googlenet.py#L163-L179

New non-detection models don't use this idiom any more (returning different output depending on jit/training flag), so I think it's safer to handle it explicitly.